### ISSUE TITLE
我居然可以在手机邮件里收到评论，妙啊

### ISSUE THREAD

**Issue by xiaziawsl** [xiaziawsl/That-is-ok#6]
[
![QQ图片2jpg](https://user-images.githubusercontent.com/78493846/106800434-68413500-669b-11eb-9dba-c0f37c3b30cd.jpg)

**Comment by ErikUden** [comment]
Yes, you can even use mobile to edit files! 您可以使用手机在GitHub上编辑文件。 您不需要电脑！

**Comment by xiaziawsl** [comment]
啊这，为什么我打不开这个图片链接，测试失败

**Comment by ErikUden** [comment]
I can see the picture!

**Comment by ErikUden** [comment]
The test succeeded!!!

**Comment by ErikUden** [comment]
This is wonderful. Your threads are great.

**Comment by xiaziawsl** [comment]
> Yes, you can even use mobile to edit files! 您可以使用手机在GitHub上编辑文件。 您不需要电脑！

really？wonderful，although I dont know how to operate

**Comment by ErikUden** [comment]
It is very complex. Don't worry. You'll figure it out. I sent you a video on WeChat!

> > Yes, you can even use mobile to edit files! 您可以使用手机在GitHub上编辑文件。 您不需要电脑！
> 
> really？wonderful，although I dont know how to operate

**Comment by ErikUden** [comment]
You changed your profile image! Congrats.

**Comment by ErikUden** [comment]
This is so funny. If you want this removed so that the entire world cannot see it, there is always an option to delete a GitHub repository, so do not worry! But I think this is funny. I hope you sleep well!

**Comment by xiaziawsl** [comment]
> You changed your profile image! Congrats.

but I can’t see your profile picture，it's weird

**Comment by ErikUden** [comment]
Really? Huh...

**Comment by ErikUden** [comment]
Reload the page? Go to my profile! very weird.... very grass

**Comment by xiaziawsl** [comment]
> Really? Huh...

maybe because the network speed is too slow，my sister‘s computer is far better than mine，so it's not a computer problem

**Comment by xiaziawsl** [comment]
> Reload the page? Go to my profile! very weird.... very grass

you really master the usage of grass

**Comment by ErikUden** [comment]
I learned from the best

**Comment by ErikUden** [comment]
> maybe because the network speed is too slow，my sister‘s computer is far better than mine，so it's not a computer problem

If it were too slow, then it simply wouldn't display the page. Not specifically stop displaying my profile picture. But regardless! It doesn't matter... since GitHub is not about pictures, GitHub is about programming, coding, and collaboration. All of this is something we'll do in the future! 

You learned so much tonight. I sent you a video on WeChat that's over 20 min long explaining the functionality of GitHub!

**Comment by ErikUden** [comment]
Try going to:
https://github.com/NotErikUden/xiazi

...and changing some of contents of the files! You should be able to do it, as you have collaboration access!

**Comment by xiaziawsl** [comment]
> > maybe because the network speed is too slow，my sister‘s computer is far better than mine，so it's not a computer problem
> 
> If it were too slow, then it simply wouldn't display the page. Not specifically stop displaying my profile picture. But regardless! It doesn't matter... since GitHub is not about pictures, GitHub is about programming, coding, and collaboration. All of this is something we'll do in the future!
> 
> You learned so much tonight. I sent you a video on WeChat that's over 20 min long explaining the functionality of GitHub!

I haven't check the wechat for a long time，so sorry！github is sooooo interesting that I'm a little excited，ummmm，everything is so new to me,I don't feel sleepy at all，，，，，，，，，，，although I SHOULD sleep

**Comment by xiaziawsl** [comment]
I've just been focusing on github ， forgetting the time

**Comment by ErikUden** [comment]
Hahahahha. yes!!!!!!!!!!!!!!!!!!!!!! You're amazing. This is exactly how I felt when going on GitHub for the first time!!! Sleep is temporary, GitHub is forever. 

(If you publish code it will be stored in an arctic vault, so it will never be lost!!)

**Comment by ErikUden** [comment]
> I've just been focusing on github ， forgetting the time

I know that feeling!

**Comment by xiaziawsl** [comment]
what is an arctic vault？

**Comment by ErikUden** [comment]
It is a storage container in the Arctic! Basically GitHub built a huge code storage container on the earth's north pole and stores all of their user's code on there so it will never get lost!

**Comment by xiaziawsl** [comment]
can you send that picture to my wechat？

**Comment by ErikUden** [comment]
Will do!
